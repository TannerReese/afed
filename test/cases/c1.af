# Test errors
Argo ^2 + Argo-1+yet = # Works fine to begin with

# Should produce missing operators error on line 5
A sentence whose words will be interpreted as variables

# Should produce missing parenthesis error on line 8
xtreme : yet + (3 - 4 * dodo))
# Should be missing operator on line 10 due to missing parenthesis
x : (yet - (( 3 * 4 ) ^ dodo   )
4 * yet = 12.800000 
# Should produce missing parenthesis error on line 13
jk : ((yet * dodo - woNder) / Argo = stuff # Comment to stop parsing

yet  : 3.2
dodo : 0.5

# Missing operators between variables
yet + dodo yet * dodo=

3 *Argo + 4.567*dodo= # The rest of the document should be fine

Argo : 2 + 3 - 1+8 - 9 // 2

# Should be missing values between operators on line 21
yet * + -dodo
Argo * woNder ^ 0.3 =

woNder : Argo * Argo

