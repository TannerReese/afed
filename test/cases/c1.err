(Line 5) PARSE_ERR_MISSING_OPERS: Multiple values without operator between
(Line 8) PARSE_ERR_PARENTH_MISMATCH: Missing open or close parenthesis
(Line 10) PARSE_ERR_MISSING_OPERS: Multiple values without operator between
(Line 21) EVAL_ERR_STACK_UNDERFLOW: Values popped when Stack empty
