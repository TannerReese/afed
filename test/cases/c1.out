# Test errors
Argo ^2 + Argo-1+yet = 74.200000 # Works fine to begin with

# Should produce missing operators error on line 3
A sentence whose words will be interpreted as variables

# Should produce missing parenthesis error on line 8
xtreme : (yet + (3 - 4 * dodo)
# Should be missing operator on line 12 due to missing parenthesis
x : (yet - (( 3 * 4 ) ^ dodo   )

4 * yet = 12.800000 
yet  : 3.2
dodo : 0.5

3 *Argo + 4.567*dodo= 26.283500 # The rest of the document should be fine

Argo : 2 + 3 - 1+8 - 9 // 2

# Should be missing values between operators on line 21
yet * + -dodo
Argo * woNder ^ 0.3 = 27.857618 

woNder : Argo * Argo

